ap/selection/test.py

```diff
@@ -269,6 +276,12 @@ def test(
     # build categories
     self.stack.category_ids(events, config_inst=config_inst, dataset_inst=dataset_inst, **kwargs)
 
+    # for cutflow plots
+    self.stack.process_ids(events, dataset_inst=dataset_inst, **kwargs)
```

Comment:
Just quick suggestion as we will always need process IDs during the selection in the future (for saving event weight sums per process id for sample stitching).

```suggestion
    # create process ids
    self.stack.process_ids(events, dataset_inst=dataset_inst, **kwargs)

```